lib/instance.js

```diff
@@ -634,10 +634,15 @@ Instance.prototype.save = function(options) {
         var instance = self.get(include.as);
         if (!instance) return Promise.resolve();
 
-        return instance.save({
+        var includeOptions = _.defaults(include.options || {}, {
           transaction: options.transaction,
           logging: options.logging
-        }).then(function () {
+        });
+        _.assign(includeOptions, {
+          parentRecord: self
+        });
```

Comment:
All that don't conflict, the goal is that the user can send any set of options to any call and have access to those specific options anywhere in the call chain where they hook in.